docs/src/main/asciidoc/security.adoc

```diff
@@ -55,35 +55,39 @@ Please see xref:security-built-in-authentication.adoc#mutual-tls[Mutual TLS Auth
+With the Quarkus OIDC extension, both `Bearer` and `Authorization Code Flow` authentication mechanisms use <<smallrye-jwt, SmallRye JWT>> to represent JWT tokens as Microprofile JWT `org.eclipse.microprofile.jwt.JsonWebToken`. 
 ====
 
-See xref:security-openid-connect-multitenancy.adoc[Using OpenID Connect Multi-Tenancy] for more information about multiple tenants which can support `Bearer` or `Authorization Code Flow` authentication mechanism and configured statically or dynamically.
+For information about `Bearer Token` authentication mechanism, see the xref:security-openid-connect.adoc[Using OpenID Connect to Protect Service Applications].
+
+For information about `Authorization Code Flow` authentication mechanism, see xref:security-openid-connect-web-authentication.adoc[Using OpenID Connect to Protect Web Application].
+
+For information about multiple tenants that can support `Bearer` or `Authorization Code Flow` authentication mechanism, see xref:security-openid-connect-multitenancy.adoc[Using OpenID Connect Multi-Tenancy].
 
 [NOTE]
 ====
-If you prefer to have Quarkus OIDC extension enabled at runtime then set `quarkus.oidc.tenant-enabled=false` at build time and re-enable it at runtime using a system property.
-See also xref:security-openid-connect-multitenancy.adoc#disable-tenant[Disabling Tenant Configurations] for more information about managing the individual tenant configurations in the multi-tenant OIDC deployments.
+If you want to enable the Quarkus OIDC extension at runtime, you set `quarkus.oidc.tenant-enabled=false` at build time then re-enable it at runtime by using a system property.
+For more information about managing the individual tenant configurations in multitenant OIDC deployments, see xref:security-openid-connect-multitenancy.adoc#disable-tenant[Disabling Tenant Configurations].
 ====
 
-If you use Keycloak and Bearer tokens then also see the xref:security-keycloak-authorization.adoc[Using Keycloak to Centralize Authorization] guide.
+If you use keycloak and bearer tokens, see the xref:security-keycloak-authorization.adoc[Using Keycloak to Centralize Authorization] guide.
 
 [NOTE]
 ====
-If you need to configure Keycloak programmatically then consider using https://www.keycloak.org/docs/latest/server_development/#admin-rest-api[Keycloak Admin REST API] with the help of the `quarkus-keycloak-admin-client` or `quarkus-keycloak-admin-client-reactive` (if the application uses `quarkus-rest-client-reactive`) extension.
+If you need to configure keycloak programmatically, then consider using https://www.keycloak.org/docs/latest/server_development/#admin-rest-api[Keycloak Admin REST API] with the help of the `quarkus-keycloak-admin-client` or `quarkus-keycloak-admin-client-reactive` (if the application uses the `quarkus-rest-client-reactive`) extension.
```

Comment:
```suggestion
If you need to configure Keycloak programmatically, then consider using https://www.keycloak.org/docs/latest/server_development/#admin-rest-api[Keycloak Admin REST API] with the help of the `quarkus-keycloak-admin-client` or `quarkus-keycloak-admin-client-reactive` (if the application uses the `quarkus-rest-client-reactive`) extension.
```

Reply:
`Keycloak` is an OpenId connect provider we recommend, so the capital `K` has to stay

Reply:
Thanks for the clarification, Sergey. I have changed this back.